src/server/index.js

```diff
@@ -20,15 +22,25 @@ export default ({ clientStats }) => async (req, res) => {
 
     const app = (
         <StaticRouter
-          location={req.url}
-          context={context}>
-            <App/>
+            location={req.url}
+            context={context}>
+            <App />
         </StaticRouter>
     );
 
     const appString = ReactDOM.renderToString(app);
+    const helmet = Helmet.renderStatic();
     const chunkNames = flushChunkNames();
-    const { js, styles, cssHash } = flushChunks(clientStats, { chunkNames });
+    const { js, styles,
+        //    cssHash 
+    } = flushChunks(clientStats, { chunkNames });
+    const document = createDocument({
+        appString,
+        js,
+        styles,
+        helmet,
+        //    cssHash,  //I'm not sure if this is needed?
```

Comment:
We can ditch cssHash, since it is not needed anymore with the current importing mechanisms